### PR TITLE
README suggestions

### PR DESCRIPTION
**Pull request by jesusvazquez**

### CODE REVIEW

**Comment by slok** [comment]
# [Codecov](https://codecov.io/gh/slok/kahoy/pull/83?src=pr&el=h1) Report
> Merging [#83](https://codecov.io/gh/slok/kahoy/pull/83?src=pr&el=desc) into [master](https://codecov.io/gh/slok/kahoy/commit/a010ee2c1891edbc99599c3c09517b6d4576955d?el=desc) will **not change** coverage.
> The diff coverage is `n/a`.

[Impacted file tree graph](https://codecov.io/gh/slok/kahoy/pull/83?src=pr&el=tree)

```diff
@@           Coverage Diff           @@
##           master      #83   +/-   ##
=======================================
  Coverage   74.97%   74.97%           
=======================================
  Files          20       20           
  Lines         839      839           
=======================================
  Hits          629      629           
  Misses        159      159           
  Partials       51       51           
```



------

[Continue to review full report at Codecov](https://codecov.io/gh/slok/kahoy/pull/83?src=pr&el=continue).
> **Legend** - [Click here to learn more](https://docs.codecov.io/docs/codecov-delta)
> `Δ = absolute <relative> (impact)`, `ø = not affected`, `? = missing data`
> Powered by [Codecov](https://codecov.io/gh/slok/kahoy/pull/83?src=pr&el=footer). Last update [a010ee2...809ae58](https://codecov.io/gh/slok/kahoy/pull/83?src=pr&el=lastupdated). Read the [comment docs](https://docs.codecov.io/docs/pull-request-comments).

File: README.md [hunk 1]
```diff
@@ -31,11 +31,15 @@ Maintain Kubernetes resources in sync easily.
 
 ## :tada: Introduction
 
-Kahoy is a minimal and flexible tool to sync/deploy your Kubernetes resource **raw** manifests and a cluster.
+Did you ever wish for a tool that could deploy all your manifests to a Kubernetes cluster without much hassle? Without worrying about applying the manifests in a specific order with kubectl, or setting up a complex CI workflow or even installing a more advanced tool that you first need to learn and observe.
```
Comment (jesusvazquez) [inline README.md:34]: Not sure about this one, I intended to frame the problem in a way that people could better imagine where Kahoy fits. I tried to do this with this parragraph and the next one, please take a look.

File: README.md [hunk 2]
```diff
@@ -31,11 +31,15 @@ Maintain Kubernetes resources in sync easily.
...
 
-Focuses on Gitops, and Kubernetes resources (not apps/releases/services/whatever) and understands git repositories.
+You probably noticed a big gap between what can be done with kubectl and more advanced tools like Helm and Spinnaker. But.. what about all the cases where we just need a bit more than what kubectl can offer?
```
Comment (slok) [inline README.md:36]:
```suggestion
You probably noticed a big gap between what can be done with kubectl and more advanced tools like Helm and Flux. But.. what about all the cases where we just need a bit more than what kubectl can offer?
```

File: README.md (hunk 1 above)
Comment (slok) [reply README.md:34]: The idea is very good, but I would make it smaller, I think is too long for an introduction, at least the first paragraph, the second (_You probably..._) is perfect.

**Review by slok — APPROVED** [review]

**Comment by jesusvazquez** [comment]
@slok I just included your suggestions, maybe you want to merge this one.

**Comment by slok** [comment]
Conflicts solved to be able to merge! thanks, @jesusvazquez :tada:

**Pull request merged**